api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -445,24 +445,18 @@ message DesktopSharedDirectoryRead {
     (gogoproto.embed) = true,
     (gogoproto.jsontag) = ""
   ];
-  // Status indicates whether the process was successful.
-  Status Status = 5 [
-    (gogoproto.nullable) = false,
-    (gogoproto.embed) = true,
-    (gogoproto.jsontag) = ""
-  ];
   // DesktopAddr is the address of the desktop being accessed.
-  string DesktopAddr = 6 [(gogoproto.jsontag) = "desktop_addr"];
+  string DesktopAddr = 5 [(gogoproto.jsontag) = "desktop_addr"];
   // DirectoryName is the name of the directory being shared.
-  string DirectoryName = 7 [(gogoproto.jsontag) = "directory_name"];
+  string DirectoryName = 6 [(gogoproto.jsontag) = "directory_name"];
```

Comment:
Changing field numbers will break protobuf compatibility.
If a field is no longer required just reserve that field number instead of changing other messages fields otherwise clients/servers cannot decode events at all